assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -504,7 +504,7 @@ che.singleport.wildcard_domain.ipless=false
 
 # Docker image of Che plugin broker app that resolves workspace tooling configuration and copies
 # plugins dependencies to a workspace
-che.workspace.plugin_broker.image=eclipse/che-plugin-broker:latest
+che.workspace.plugin_broker.image=eclipse/che-plugin-broker:19d258b
```

Comment:
maybe it should have more meaningful tags bounded to versions? like v0.0.1 etc. WDYT?

Reply:
In progress https://bugs.eclipse.org/bugs/show_bug.cgi?id=537921#c4

Reply:
AFAIK I did it with a github file https://github.com/ws-skeleton/che-fortune-plugin/blob/master/hooks/post_push